Sources/PT.PM.Common/Nodes/GeneralScope/TypeDeclaration.cs

```diff
@@ -12,23 +12,23 @@ public class TypeDeclaration : EntityDeclaration
 
         public List<TypeToken> BaseTypes { get; set; } = new List<TypeToken>();
 
-        public List<EntityDeclaration> TypeMembers { get; set; } = new List<EntityDeclaration>();
+        public List<Ust> TypeMembers { get; set; } = new List<Ust>();
```

Comment:
Why do you replace `EntityDeclaration` with `Ust` here?

Reply:
In some cases we have `UsingDeclaration` in `TypeDeclaration`. `UsingDeclaration` is not `EntityDeclaration`